### ISSUE TITLE
ansible-lint failes with reboot module and python 3.7

### ISSUE THREAD

**Issue by mhubig** [ansible/ansible-lint#395]
# Issue Type
- Feature request

# Ansible and Ansible Lint details

```shell
$ ansible --version
ansible 2.7.1
  config file = None
  configured module search path = ['/Users/d0287062/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.7.1/libexec/lib/python3.7/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.7.1 (default, Nov  6 2018, 18:46:03) [Clang 10.0.0 (clang-1000.11.45.5)]
$ ansible-lint --version
ansible-lint 3.4.23
```

- ansible installation method: brew
- ansible-lint installation method: brew

# Desired Behaviour

ansible-lint should not report the new [reboot module](https://docs.ansible.com/ansible/2.7/modules/reboot_module.html) as error.

# Actual Behaviour (Bug report only)

Given this playbook:
```yml
- hosts: all
  tasks:
    - name: Restarting the system
      reboot:
```

I get this error:
```shell
$ ansible-lint test.yml
Couldn't parse task at test.yml:3 (no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '<unicode string>': line 3, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

(could not open file to display line))
{'name': 'Restarting the system', 'reboot': None}
```

**Comment by gundalow** [comment]
This works as expected for me
```
cat ansible-lint-395-reboot.yml
- hosts: all
  tasks:
    - name: Restarting the system
      reboot:
```


````
ansible-lint -vvv ansible-lint-395-reboot.yml
Examining ansible-lint-395-reboot.yml of type playbook
```

What's on line 3 column 7, that unicode string looks like it might be causing issues

**Comment by mhubig** [comment]
There is no unicode within the `test.yml` file. What version of python are you using? In 3.x every string is byte or unicode, so the error msg may be misleading ...

**Comment by mhubig** [comment]
I can confirm. Test with python 2.7  => works!

**Comment by chaosblog** [comment]
The problem exists for me Python 2.7.12 and ansible-lint 4.1.0. 
```
Couldn't parse task at test.yml:40 (no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '<unicode string>': line 40, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

(could not open file to display line))
{ u'name': u'reboot server', u'reboot': None}
```

The code in the playbook is:
```
- name: "reboot server"
  reboot:
```

**Comment by claco** [comment]
I have the same issue in 2.7.16:

`Couldn't parse task at /Users/claco/Projects/claco.ansible/roles/claco.apt/tasks/ubuntu.yml:31 (no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '<unicode string>': line 31, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

(could not open file to display line))
{ u'name': u'Reboot To Complete Upgrades',
  u'reboot': { '__file__': u'/Users/claco/Projects/claco.ansible/roles/claco.apt/tasks/ubuntu.yml',
               '__line__': 33,
               u'msg': u'Rebooting system to complete upgrades'},
  u'tags': u'apt-reboot',
  u'when': u'apt_reboot'}
make: *** [lint-ansible-roles] Error 1
`

for this task:

`- name: Reboot To Complete Upgrades
  reboot:
    msg: "Rebooting system to complete upgrades"
  when: apt_reboot
  tags: apt-reboot
`

Ansible 2.8.3 and ansible-lint 4.1.0

**Comment by npklein** [comment]
I have the same issue, when I do ansible-lint on the specific file with 

    ansible-lint tasks/gitclone.yml

I don't get an error, but when I do

   ansible-lint .

I get 

```
Couldn't parse task at /Users/NPK/UMCG/git_projects/molgenis-app-genenetwork/ansible/vagrant/tasks/gitclone.yml:2 (no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '<unicode string>': line 2, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

(could not open file to display line))
{ 'hosts': 'all',
  'remote_user': 'root',
  'tasks': [ { '__file__': '/Users/NPK/UMCG/git_projects/molgenis-app-genenetwork/ansible/vagrant/tasks/gitclone.yml',
               '__line__': 6,
               'git': { '__file__': '/Users/NPK/UMCG/git_projects/molgenis-app-genenetwork/ansible/vagrant/tasks/gitclone.yml',
                        '__line__': 8,
                        'dest': '/opt/repo',
                        'repo': 'git@github.com:npklein/molgenis-app-genenetwork.git',
                        'version': 'metabrainnetwork'},
               'name': 'Clone a metabrainnetwork repository into /opt.'}]}
```

**Comment by pescobar** [comment]
I was also hitting this bug with ansible-lint 4.1.0 and python 3.6

I have workarounded it by now doing `pip install ansible-lint==4.0.0` in my CI config

**Comment by qume** [comment]
Hitting this with Python 2.7.15rc1 and ansible-line 4.1.0 and 4.0.0

**Comment by rawat-he** [comment]
Hit with same issue with
```
ansible-lint 4.2.0
Python 3.6.8
```
but ```ansible-lint 4.3.4``` has no issues

**Comment by ssbarnea** [comment]
You file is invalid, failure is very clear implicit "None" is not acceptable input for Ansible. Please used discussions https://github.com/ansible/ansible-lint/discussions for support.
